package.json

```
@@ -10,6 +10,7 @@
     "check-format": "turbo run check-format",
     "cspell": "cspell --no-progress .",
     "format": "turbo run format",
+    "install-conflict": "git fetch --all && git checkout upstream/main -- pnpm-lock.yaml && pnpm install",
```

or just fetch upstream
```suggestion
    "install-conflict": "git fetch upstream && git checkout upstream/main -- pnpm-lock.yaml && pnpm install",
```